src/proc_image_processing/cpu/filters/detectors/vampire_torpedoes_detector.h

```diff
@@ -80,64 +80,70 @@ namespace proc_image_processing {
                     cv::RotatedRect box = cv::fitEllipse(pointfs);
 
                     float circleIndex;
-                    float percentageFilled;
+                    float percentFilled;
 
                     if (look_for_ellipse_()) {
 
-                        circleIndex = CalculateCircleIndex(contours[i]);
+                        circleIndex = getCircleIndex(contours[i]);
 
                         //std::cout << circleIndex << std::endl;
 
                         if (circleIndex < 0.7) {
                             continue;
                         }
 
-                        percentageFilled = CalculatePercentFilled(output_image_, box);
+                        percentFilled = getPercentFilled(output_image_, box);
 
-                        if (percentageFilled > 50) {
+                        if (percentFilled > 50) {
                             continue;
                         }
 
                         if (debug_contour_()) {
                             cv::drawContours(output_image_, contours, i, CV_RGB(0, 255, 0), 2);
                         }
 
-                        objectif = "vampire_torpedoes";
+                        objective = "vampire_torpedoes";
                     }
 
                     if (look_for_heart_()) {
 
-                        circleIndex = CalculateCircleIndex(contours[i]);
+                        circleIndex = getCircleIndex(contours[i]);
 
                         if (circleIndex > 0.9) {
                             continue;
                         }
 
-                        percentageFilled = CalculatePercentFilled(output_image_, box);
+                        percentFilled = getPercentFilled(output_image_, box);
 
-                        if (percentageFilled > 50) {
+                        if (percentFilled > 50) {
                             continue;
                         }
 
                         if (debug_contour_()) {
                             cv::drawContours(output_image_, contours, i, CV_RGB(0, 255, 0), 2);
                         }
 
-                        objectif = "heart_torpedoes";
+                        objective = "heart_torpedoes";
                     }
                     objVec.push_back(object);
                 }
 
-                std::sort(objVec.begin(), objVec.end(), [](ObjectFullData::Ptr a, ObjectFullData::Ptr b) -> bool { return a->GetArea() > b->GetArea();});
+                std::sort(objVec.begin(), objVec.end(),
+                          [](const ObjectFullData::Ptr &a, const ObjectFullData::Ptr &b) -> bool {
+                              return
```

Comment:
This formatting looks overkill for the situation xd

Reply:
It busts the 120 char line limit